### DISCUSSION TITLE
Does AWS Open search Connection only work with VPC endpoint now?

### DISCUSSION THREAD

**Discussion by Yogeshberia** [jaegertracing/jaeger#4694] [Q&A] [answered]
Self-explanatory, does the AWS ES/open search connection only work if there is a VPC endpoint?

We used to have a setup earlier where it also used to work with the domain endpoint with the access key and secret, where the policy didn't restrict access.
I want to know if this is even possible or I am just wasting my time trying to get this working.

**Comment by jkowall** [comment, marked as answer]
Jaeger supports OpenSearch and ElasticSearch as a backend (datastore), there is nothing specific in Jaeger to limit how it connects, hence the use of VPCs is up to you how you manage the network.
